packages/ek-components/src/components/EkContentCard.vue

```diff
@@ -3,14 +3,14 @@
     noBody
     class="my-2 rounded-lg"
     :class="{
-      'shadow-sm': !isHovered && node.available,
-      'shadow': isHovered && node.available,
-      'disabled': !node.available,
+      'shadow-sm': !isHovered && !isDisabled,
+      'shadow': isHovered && !isDisabled,
```

Comment:
```suggestion
      'shadow-sm': !isHovered && isEnabled,
      'shadow': isHovered && isEnabled,
```
This makes it a bit easier to read (IDK if there is a way to avoid the `!` for `isHovered` as well).

Reply:
Good catch!